syncer/checkpoint.go

```diff
@@ -473,6 +476,18 @@ func (cp *RemoteCheckPoint) GlobalPoint() mysql.Position {
 	return cp.globalPoint.MySQLPos()
 }
 
+// ShardTablePoint implements CheckPoint.ShardTablePoint
+func (cp *RemoteCheckPoint) ShardTablePoint() map[string]map[string]mysql.Position {
+	tablePoints := make(map[string]map[string]mysql.Position)
```

Comment:
1. Do we need to `RLock` here? Is `cp.points` possible to be changed in any other calls?
2. Is this `func` return all tables' checkpoint rather than just _shard_ tables?

Reply:
done in 6252cf5